### PR TITLE
image redactor: initialize AnalyzerEngine instance only once

### LINKED ISSUE #645: Image anonymization is slow
Hi,

First of all, thank you guys for making this awesome project :)

I wanted to ask if there are any ways to improve image anonymization performance. I'm using Presidio as a python package and it takes about 5~ seconds to process pretty small images. I'm using the example code from the Presidio docs.

### PR DESCRIPTION
**Pull request by kubistika**
Hopefully this should fix #645.

### CODE REVIEW

**Comment by omri374** [comment]
Thanks @kubistika! looks great.

File: presidio-image-redactor/presidio_image_redactor/image_analyzer_engine.py [hunk 1]
```diff
@@ -11,6 +11,9 @@
 class ImageAnalyzerEngine:
     """ImageAnalyzerEngine class."""
 
+    def __init__(self):
```
Comment (omri374) [inline presidio-image-redactor/presidio_image_redactor/image_analyzer_engine.py:14]:
```suggestion
    def __init__(self, analyzer_engine: AnalyzerEngine = None):
        if not analyzer_engine:
            self.analyzer_engine = AnalyzerEngine()
        else:
            self.analyzer_engine = analyzer_engine
```

File: presidio-image-redactor/presidio_image_redactor/image_analyzer_engine.py [hunk 2]
```diff
@@ -11,6 +11,9 @@
 class ImageAnalyzerEngine:
     """ImageAnalyzerEngine class."""
```
Comment (omri374) [inline presidio-image-redactor/presidio_image_redactor/image_analyzer_engine.py:12]:
```suggestion
    """ImageAnalyzerEngine class.

    :param analyzer_engine: The Presidio AnalyzerEngine instance
        to be used to detect PII in text.
    """
```

**Comment by kubistika** [comment]
@omri374 Thanks for the review.
I think my changes are already included in #651.

Have a great day :)

**Comment by omri374** [comment]
@kubistika Agree. I pushed #651 before seeing you proposed this PR, so I'd be happy to merge yours and then #651.

**Comment by kubistika** [comment]
Ok, thanks :)

File: presidio-image-redactor/presidio_image_redactor/image_analyzer_engine.py [hunk 3]
```diff
@@ -9,7 +9,16 @@
...
-    """ImageAnalyzerEngine class."""
+    """ImageAnalyzerEngine class.
+    :param analyzer_engine: The Presidio AnalyzerEngine instance
+        to be used to detect PII in text.
+    """
+
+    def __init__(self, analyzer_engine: AnalyzerEngine = None):
+        if not analyzer_engine:
+            self.analyzer_engine = AnalyzerEngine()
+        else:
```
Comment (shiranr) [inline presidio-image-redactor/presidio_image_redactor/image_analyzer_engine.py:20]:
```suggestion
        if not analyzer_engine:
            analyzer_engine = AnalyzerEngine()
        self.analyzer_engine = analyzer_engine    
```

**Comment by kubistika** [comment]
Fixed a missing blank line (linting error, sorry).

**Review by SharonHart — APPROVED** [review]

**Review by navalev — APPROVED** [review]

**Pull request merged**